lib/ProductOpener/Users.pm

```diff
@@ -949,6 +969,27 @@ sub remove_user_by_org_admin ($orgid, $user_id) {
 	return;
 }
 
+sub add_users_to_org_by_admin ($org_id, $email_list) {
```

Comment:
Maybe the function could return an object with 2 lists, 1 for e-mails that have been added, and another for e-mails that have been invited.